docs/src/api.md

```diff
@@ -36,25 +36,53 @@ isoper
 isoperbra
 isoperket
 issuper
+LinearAlgebra.ishermitian
+isherm
+LinearAlgebra.issymmetric
+LinearAlgebra.isposdef
 ```
 
 ## [Qobj arithmetic and attributes](@id doc-API:Qobj-arithmetic-and-attributes)
 
 ```@docs
-sqrtm
-sinm
-cosm
+Base.conj
+LinearAlgebra.transpose
+LinearAlgebra.adjoint
+LinearAlgebra.dot
+LinearAlgebra.sqrt
+LinearAlgebra.log
+LinearAlgebra.exp
 LinearAlgebra.tr
 LinearAlgebra.svdvals
 LinearAlgebra.norm
+LinearAlgebra.normalize
+LinearAlgebra.normalize!
+LinearAlgebra.inv
+LinearAlgebra.diag
+sinm
+cosm
+proj
 ptrace
+purity
+permute
 tidyup
 tidyup!
 get_data
 get_coherence
 partial_transpose
 ```
 
+## [Synonyms for Qobj attribute functions](@id doc-API:Synonyms-for-Qobj-attribute-functions)
+```@docs
```

Comment:
I think that also `isherm` should be moved here

Reply:
sure, thanks for the reminder